pkg/cmd/pipelinerun/log_reader.go

```diff
@@ -121,11 +129,35 @@ func (lr *LogReader) readAvailableLogs(pr *v1alpha1.PipelineRun) (<-chan Log, <-
 
 			pipeLogs(logC, errC, tlr)
 		}
+		if pr.Status.Conditions[0].Status == corev1.ConditionFalse {
+			errC <- fmt.Errorf(pr.Status.Conditions[0].Message)
+		}
 	}()
 
 	return logC, errC, nil
 }
 
+func (lr *LogReader) waitUntilAvailable(timeout int, limit int) {
```

Comment:
Could we prefer the `PipelineRun` watch API instead of this?  :thinking:

Reply:
Thanks for pointing, using watch api now.

Reply:
:+1: